src/NodeDev.Blazor/Components/ProjectToolbar.razor

```diff
@@ -0,0 +1,81 @@
+﻿@using Microsoft.AspNetCore.Components.Forms
+@using NodeDev.Blazor.Services
+@using NodeDev.Core
+@inject ProjectService ProjectService
+@inject ISnackbar Snackbar
+
+<MudFileUpload T="IBrowserFile" FilesChanged="Open">
+    <ActivatorContent>
+        <MudButton Class="ml-3" Disabled=@Project.IsLiveDebuggingEnabled data-test-id="openProject">Open Project</MudButton>
+    </ActivatorContent>
+</MudFileUpload>
+<MudButton OnClick="NewProject" Class="ml-3" Disabled=@Project.IsLiveDebuggingEnabled data-test-id="newProject">New Project</MudButton>
+<MudButton OnClick="Save" Class="ml-3" data-test-id="Save">Save</MudButton>
+<MudButton OnClick="Add" Class="ml-3">Add node</MudButton>
+<MudButton OnClick="Run" Class="ml-3">Run</MudButton>
+<MudButton OnClick="SwitchLiveDebugging">@(Project.IsLiveDebuggingEnabled ? "Stop Live Debugging" : "Start Live Debugging")</MudButton>
+<MudSpacer />
+<MudIconButton Icon="@Icons.Material.Filled.MoreVert" Color="Color.Inherit" Edge="Edge.End" />
+
+
+@code {
+
+    private Project Project => ProjectService.Project;
+
+    [Parameter]
+    public EventCallback OnProjectChanged { get; set; }
+
+    private IBrowserFile? File;
```

Comment:
I don't think this is actually used, we can remove it and use 'file' locally in the Open method